ros2cli/package.xml

```diff
@@ -9,6 +9,7 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <exec_depend>python3-pkg-resources</exec_depend>
```

Comment:
I thought that `ament_python` packages rely on setuptools at buildtime that provide this dependency. So I assumed that runtime would be enough, isn't it ?